index.js

```diff
@@ -232,7 +232,7 @@ class ServerlessS3Sync {
             ...contentTypeObject,
             ...file.params,
             ...{
-              CopySource: file.name.replace(localDir, `${s.bucketName}${bucketPrefix == '' ? '' : bucketPrefix}/`),
+              CopySource: file.name.replace(localDir + path.sep, `${s.bucketName}${bucketPrefix == '' ? '' : bucketPrefix}/`),
```

Comment:
I have left the trailing slash after the bucket prefix to conform with AWS expectations.  To allow for this I added the path.sep (operating system specific) on the original file name